src/mavsdk/core/mavlink_request_message_handler.cpp

```diff
@@ -0,0 +1,126 @@
+#include "mavlink_request_message_handler.h"
+#include "system_impl.h"
+
+namespace mavsdk {
+
+MavlinkRequestMessageHandler::MavlinkRequestMessageHandler(SystemImpl& system_impl) :
+    _system_impl(system_impl)
+{
+    _system_impl.register_mavlink_command_handler(
+        MAV_CMD_REQUEST_MESSAGE,
+        [this](const MavlinkCommandReceiver::CommandLong& command) {
+            return handle_command_long(command);
+        },
+        this);
+
+    _system_impl.register_mavlink_command_handler(
```

Comment:
Is the duplicate register_mavlink_command_handler on purpose?

Reply:
Ooops, good catch! I think one should be for command_int!